lib/actions-oclif/env/rm.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright 2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command, flags } from '@oclif/command';
+import { stripIndent } from 'common-tags';
+
+import { CommandHelp } from '../../utils/oclif-utils';
+
+interface FlagsDef {
+	device: boolean;
+	yes: boolean;
+}
+
+interface ArgsDef {
+	id: number;
+}
+
+export default class EnvRmCmd extends Command {
+	public static description = stripIndent`
+		Remove an environment variable from an application or device.
+
+		Remove an environment variable from an application or device, as selected
+		by the respective command-line options.
```

Comment:
Minor: the word "respective" above applies to "env add" but not so much to "env rm", because "env add" has the "--application" and "--device" options, whereas "env rm" only has "--device". Perhaps:
```
		Remove an environment variable from an application or device, as selected
		by command-line options.
```

Reply:
Yep, ideal